css-box-model/css-box-model-quiz-notes.md

```diff
@@ -0,0 +1,38 @@
+# css-box-model-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What CSS properties make up the box model?
+borders, margin,  padding, width, height, background-color, display, visibility
```

Comment:
After our Q&A take a shot at answering this question again. There are a few properties that shouldn't be here.